libs/langgraph/langgraph/pregel/main.py

```diff
@@ -636,7 +636,10 @@ def __init__(
         name: str = "LangGraph",
         **deprecated_kwargs: Unpack[DeprecatedKwargs],
     ) -> None:
-        if config_type := deprecated_kwargs.get("config_type"):
+        if (
+            config_type := deprecated_kwargs.get("config_type"),
+            MISSING,
+        ) is not MISSING:
```

Comment:
@sydney-runkle Is there a bug here? it looks as though this is testing something roughly akin to `(config_type, MISSING) is not MISSING`, that is, a comparison of tuple with `is not MISSING`